README.md

```diff
@@ -18,20 +18,31 @@ npm install
 
 Running `npm install` will install the NPM dependencies.
 
-Then install the Python dependencies:
+Create a virtual environment and install install the Python dependencies:
 
 ```
+python -m venv venv
```

Comment:
*Question*: should this be `python3` or `python`? Not sure since I saw [this](https://mozilla.slack.com/archives/G7ACJHJCB/p1678988266531079?thread_ts=1678987708.882689&cid=G7ACJHJCB) in our team slack channel

Reply:
Hm, I don't know either. I assumed there was a problem with my environment that the basic `python` command didn't work. @pmac do you know what is most likely to work out of the box?

Reply:
may not work out of the box unless you have installed python via homebrew or pyenv.

Reply:
K. I'm going to merge because I don't want to agonize over this and more docs is better.